ntoskrnl/ntos.cmake

```diff
@@ -275,13 +275,19 @@ list(APPEND SOURCE
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/accesschk.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/acl.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/audit.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/client.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/objtype.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/priv.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/sd.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/semgr.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/sid.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/sqos.c
     ${REACTOS_SOURCE_DIR}/ntoskrnl/se/srm.c
-    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/token.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/subject.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/tknadj.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/tkncls.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/tknlife.c
+    ${REACTOS_SOURCE_DIR}/ntoskrnl/se/tknsup.c
```

Comment:
looks like these "tkn*" files' names are short enough so that you could use "token" in their names instead (making them a bit more readable), without going over the 8.3 length :)